projects/core/src/routing/configurable-routes/url-translation/product-url.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { SemanticPathService } from './semantic-path.service';
+import { ProductURLCommand } from './productURL-command';
+import { Product } from '../../../model/product.model';
+@Pipe({
+  name: 'cxProductUrl',
+})
+export class ProductURLPipe implements PipeTransform {
+  constructor(private semanticPath: SemanticPathService) {}
+  transform(item: Product) {
+    return this.semanticPath.transform(new ProductURLCommand(item));
+  }
```

Comment:
we can do this:
```
transform(product: Product) {
    return this.semanticPath.transform({cxRoute: 'product', params: product});
 }
```
So, no need to create `ProductURLCommand`.

Reply:
hi

i made the requested changes. 
Thanks
Reyhaneh